pkg/tools/etcd_tools.go

```diff
@@ -418,20 +418,24 @@ func (h *EtcdHelper) AtomicUpdate(key string, ptrToType runtime.Object, ignoreNo
 	}
 }
 
-func checkEtcd(host string) error {
+// Performs a version check against the provided Etcd server, returning a triplet
```

Comment:
The comment should start with the name of the function, e.g. "CheckEtcd performs ..."